### PR TITLE
CONTRIBUTING.md: Add a new standard for PRs title

### PR DESCRIPTION
**Pull request by ShellyGolden**
A new standard for PRs title: <Component name>:  explanation in a few words
related issue: #44

### CODE REVIEW

**Comment by ShellyGolden** [comment]
I have a little problem, I'll take care of tomorrow

**Comment by adiabramovitch** [comment]
The title is deceiving, I would split this PR into two. 
One regarding the CONTRIBUTING file and the other one for the rest of the commits.

**Comment by ShellyGolden** [comment]
#44

**Comment by ShellyGolden** [comment]
> The title is deceiving, I would split this PR into two. One regarding the CONTRIBUTING file and the other one for the rest of the commits.

I have a problem and I would be glad if you help me. 
When I opened that PR with the last commit, it added the first five commits and I don't know why...
How could I fix it? What do I need to do so it won't happen again in the future?

**Comment by adiabramovitch** [comment]
You should create a different **clean** branch based on the main branch and then add your changes there. 
in the meantime, you can use interactive rebase to remove the non-related commit 
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

File: expenses/tests_expenses.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+@pytest.fixture
+def expense_test0():
+    return Expenses(amount=AMOUNT,
+                    date=DATE,
+                    Category=CATEGORY
+                    )
+
+
+@pytest.mark.django_db
+class TestExpenseדModel:
```
Comment (ronyyosef) [inline expenses/tests_expenses.py:19]: There is a "ד" in the middle of the name?

File: expenses/models.py [hunk 2]
```diff
@@ -23,4 +28,9 @@ class Category(models.TextChoices):
     )
 
     def __str__(self):
+        return f'Category:{self.category},Amount:{self.amount}'
+
+    def recent(self):
```
Comment (ronyyosef) [inline expenses/models.py:33]: Recent should be removed if it's not needed or add the logic and tests for that

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ After the dependencies were installed and the VM initialized, you can proceed wo
 # Pull Request Standards 
  - Each Pull Request should focus on single responsibility principle.
  - **The Pull Request should not break any of the existing functionality**
+ - PRs title: <Component name>:  explanation in a few words
```
Comment (ronyyosef) [inline CONTRIBUTING.md:40]: Is it the right place for changing the CONTRIBUTING.md file?

**Review by ronyyosef — COMMENTED** [review]
Great work
I left some minor comments

File: CONTRIBUTING.md (hunk 3 above)
Comment (shlior7) [reply CONTRIBUTING.md:40]: what do you mean? you have a different place in mind?

**Pull request closed without merging**